### DISCUSSION TITLE
What does `:terminal!` do?

### DISCUSSION THREAD

**Discussion by adigitoleo** [neovim/neovim#28547] [Q&A] [answered]
What does `:terminal!` do compared to `:terminal`? The bang attribute is not documented, despite being allowed and shown as an optional attribute in the command signature in `:h :terminal`.

**Comment by justinmk** [comment, marked as answer]
Similar to `:edit!`. But that only matters for 'nohidden'.

**Comment by adigitoleo** [comment]
Thanks, `:h 'hidden'` has it indeed:

```
Commands that move through the buffer list sometimes hide a buffer
although the 'hidden' option is off when these three are true:
- the buffer is modified
- 'autowrite' is off or writing is not possible
- the '!' flag was used
```
